SF iOS/SF iOS/Helpers/UserNotification/BackgroundFetchScheduler.m

```diff
@@ -0,0 +1,88 @@
+//
+//  BackgroundFetchScheduler.m
+//  Coffup
+//
+//  Created by Zachary Drayer on 6/3/19.
+//
+
+#import "BackgroundFetchScheduler.h"
+#import "GroupBackgroundFetcher.h"
+
+#if defined(__IPHONE_13_0)
+@import BackgroundTasks;
+#endif
+
+@import UIKit;
+
+static NSString *const BackgroundTaskGroupUpdateIdentifier = @"BackgroundTaskGroupUpdateIdentifier";
+
+@interface BackgroundFetchScheduler ()
+
+@property (nonatomic) GroupBackgroundFetcher *bgFetcher;
+
+@end
+
+@implementation BackgroundFetchScheduler
+
+- (void)schedule
+{
+#if defined(__IPHONE_13_0)
+	if (@available(iOS 13.0, *)) {
```

Comment:
why did we need the double check here?

Reply:
the first #if allows us to keep compiling with Xcode 10, the second is the API guard for backwards compatibility when running on iOS 12 or under